build-support/bin/run_scoot_integration.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+set -euxo pipefail
+
+export GOPATH="${GOPATH:-$(readlink -f ../go)}"
+
+clone_dir="${GOPATH}/src/github.com/twitter/scoot"
+
+if [[ ! -d "$clone_dir" ]]; then
+  mkdir -p ../go/src/github.com/twitter
+  mkdir -p ../go/bin
+  git clone https://github.com/cosmicexplorer/scoot "$clone_dir"
+  pushd "$clone_dir"
+  git checkout make-scoot-run-locally-for-pants
+  popd
+fi
+
+pushd "$clone_dir"
+
+bash ./get_fs_util.sh
+
+dep ensure -update
+
+scoot_logs="$(pwd)/scoot-logs"
+
+go run ./binaries/setup-cloud-scoot/main.go --strategy local.local &>"$scoot_logs" &
+
+# Wait for the local cluster to start up a bit before trying to start pants on it.
+sleep 15
```

Comment:
Another reason to make this into a real integration test!

Reply:
Done, by scanning the output of the subprocess for the correct ports to use in an integration test! The test itself still fails, possibly because the local cluster disconnects after we close stdout, but this synchronization part is at least fixed.